h/claim/util.py

```diff
@@ -0,0 +1,12 @@
+def generate_claim_token(request, user_id):
+    return request.registry.claim_serializer.dumps({'user_id': user_id})
```

Comment:
I think we do 'userid' instead of 'user_id' in most places throughout the code. Let's pick that and stick to it.